server/libs/Controller.php

```diff
@@ -136,7 +136,7 @@ public function uploadFile($forceUpload = false) {
         $fileUploader = FileUploader::getInstance();
         $fileUploader->setMaxSize($maxSize);
 
-        if($fileUploader->upload('file')) {
+        if($fileUploader->uploadCSVFile('file')) {
```

Comment:
Oops. This one should remain a normal `upload`, otherwise when trying to upload PDFs or something else on other parts of the system, you would receive an error.